person.rb

```diff
@@ -23,6 +24,10 @@ def correct_name
     @name
   end
 
-  attr_reader :id
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] @haliljon in the creation of a new rental we also need to pass a date parameter,
`hint`
``` suggestion
 def add_rental(parameters needed)
     Rental.new(parameters)
 end
```

Reply:
@danmainah Thank you for the hint 👍